src/Dfc.CourseDirectory.WebV2/Content/utilities/table.css

```diff
@@ -1,3 +1,7 @@
 ﻿.pttcd-table--last-row-border-0 tbody tr:last-child td {
     border: 0;
+}
+
+.table-header-width__40 th {
```

Comment:
`.pttcd-\!-width-40` in `width.css`?

Reply:
It would need to be added to every `<th>` in the file for that to work, which I feel is excessive. I've prefixed it with `pttcd-...`.

Reply:
Totally missed the th